README.md

```diff
@@ -1,5 +1,8 @@
 # Cordentity
 
+**Note:** This is a **deprecated** of the formal [Hyperledger Labs](https://github.com/hyperledger-labs/cordentity) repository,
```

Comment:
Formal - official?

Reply:
copy-paste from Hyperledger Fabric, can be changed on "official". It makes sense.

Reply:
Grammar mistake: This is a **deprecated** (what - clone or version or mirror) of the official repo